src/FffGcodeWriter.cpp

```diff
@@ -817,6 +817,17 @@ void FffGcodeWriter::addMeshLayerToGCode(SliceDataStorage& storage, SliceMeshSto
     }
     part_order_optimizer.optimize();
 
+    EFillMethod infill_pattern = mesh->getSettingAsFillMethod("infill_pattern");
+    if ((infill_pattern == EFillMethod::LINES || infill_pattern == EFillMethod::ZIG_ZAG) && mesh->infill_angles.size() == 0)
```

Comment:
Hmmm, wait.

For all infill types other than lines and zig zag it would only be beneficial to supply a single angle, though. Multiple angles would distort the pattern.

I could see a use for triangles with `{0, 90}` infill angles, though.